server/request_sendrawtx.go

```diff
@@ -69,6 +69,13 @@ func (r *RpcRequest) handle_sendRawTransaction() {
 	// Check if transaction needs protection
 	needsProtection := r.doesTxNeedFrontrunningProtection(r.tx)
 
+	// If users specify a bundle ID, cache this transaction
+	if cacheId := r.req.URL.Query().Get("bundle-id"); cacheId != "" {
```

Comment:
We should rename `bundle-id` to something better - let's discuss this in Discord

Reply:
After some more thinking, how about just `?bundle=123`?